src/net/server/mod.rs

```diff
@@ -533,6 +540,22 @@ mod e2e_tests {
         assert_eq!(expected, resp_body);
     }
 
+    struct NoVerify;
+
+    impl ServerCertVerifier for NoVerify {
+        fn verify_server_cert(
+            &self,
+            _end_entity: &Certificate,
+            _intermediates: &[Certificate],
+            _server_name: &ServerName,
+            _scts: &mut dyn Iterator<Item = &[u8]>,
+            _ocsp_response: &[u8],
+            _now: SystemTime,
+        ) -> Result<ServerCertVerified, rustls::Error> {
+            Ok(ServerCertVerified::assertion())
```

Comment:
Don't we need dangerous_configuration enabled in rustls for this to work?

Reply:
that only brings a special `DangerousClientConfig` with `set_certificate_verifier` only. does not seem to be worth it

Reply:
`axum-server` enables it. (Not saying that is right or that we should rely on it, but that's what's going on.)

Reply:
eh, true I missed that. let me be explicit about it in Cargo.toml

```
#[allow(unreachable_pub)]
#[derive(Debug)]
#[cfg_attr(docsrs, doc(cfg(feature = "dangerous_configuration")))]
pub struct ServerCertVerified(());
```